pom.xml

```diff
@@ -22,7 +22,7 @@
 
     <groupId>de.bdr.servko</groupId>
     <artifactId>gematik-idp</artifactId>
-    <version>1.1.0</version>
+    <version>1.2.0-SNAPSHOT</version>
```

Comment:
It should be version 1.2.0  without SNAPSHOT

Reply:
Yes, you're right. I've adapted it.